flox-bash/lib/commands/general.sh

```diff
@@ -224,21 +224,32 @@ function floxSearch() {
 	: "${GREP_COLOR:=1;32}"
 	export GREP_COLOR
 
+	if [[ "$showDetail" = true ]]; then
+		_m_col="cat -"
+	else
+		_m_col="$_column -s '|' -t -l 2"
```

Comment:
I get the following error when running:

```
$ flox build flox
$ ./result/bin/flox search python
Migrating previous telemetry opt out to user config
/nix/store/4vwsj74yv5vzwdzxrvl6rc3gj1bd75q1-util-linux-1003.1-2008/bin/column: illegal option -- l
Usage: column [-tx] [-c columns] [-s sep] [file ...]
warning: input 'flox' has an override for a non-existent input 'nixpkgs-flox'
warning: input 'flox' has an override for a non-existent input 'nixpkgs-flox'
warning: input 'flox' has an override for a non-existent input 'nixpkgs-flox'
```

Reply:
Good catch, this is a difference between the FreeBSD and GNU implementations of `column`.
I'll get it patched.